README.md

```diff
@@ -13,6 +15,20 @@ The network will do a quick start, and everyone can join in later by creating a
 
 ## Getting started
 
+### Upgrading from genki-4000
+
+We are attempting a hard fork from `genki-4000` on December 28th.
+
+if you were validating on `genki-4000`,
+
+1. Halt your node.
+2. Consider backing up `$HOME/.gaiad/data`. This will make it possible to restart `genki-4000` if the hard fork state is invalid.
+3. Download the new genesis file from `https://raw.githubusercontent.com/certusone/genki-4000/master/genesis-final.json`
```

Comment:
Genki evolution has been following a pattern  of create a new repo, not to PR over a previous genesis (creating inconsistency).  Maybe a new file in a genki-4001 directory is better?

Reply:
The new repo was mostly to make collecting gen-txs simple. We don't need to do that in this case.